icons/cheese.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M9.8 4.2a1.88 1.88 0 0 1-2.6 2.6L3 11v9c0 .6.4 1 1 1h3a2 2 0 0 1 4 0h9c.6 0 1-.4 1-1v-9s-1-6-9-9Z" />
+  <path d="M3 12h8" />
+  <path d="M17 12c0-1.1-1.3-2-3-2s-3 .9-3 2a3 3 0 1 0 6 0" />
+  <path d="M21 12h-4" />
+  <circle cx="5" cy="14" r="2" />
+  <circle cx="19" cy="19" r="2" />
```

Comment:
There are some violations of the 2px rule here.

Reply:
> There are some violations of the 2px rule here.

@jguddas Which parts, specifically?

<img width="498" alt="Untitled" src="https://github.com/lucide-icons/lucide/assets/7797479/cf97dfb5-7afc-4fdb-af8c-c27a898e05c5">

Reply:
![image](https://github.com/lucide-icons/lucide/assets/25524993/8664eae8-36c1-4087-bbfb-39c51184a11b)